script/libtest.bash

```diff
@@ -52,9 +52,10 @@ k8s_wait_for_pod_ready() {
 }
 k8s_wait_for_uniq_pod() {
     echo_info "Waiting for pod '${@}' to be the only one running ... "
-    local -i cnt=360
+    local -i cnt=${TEST_MAX_WAIT_SEC:?}
     kubectl get pod "${@}"
-    until [ `kubectl get pod "${@}" | grep 1/1 | wc -l | awk '{print $1}'` -eq "1" ] ; do
+    until [ `kubectl get pod "${@}" | wc -l | awk '{print $1}'` -eq "1" ] ; do
```

Comment:
Please use
```
[[ $(kubectl get pod "${@}" | grep -c 1/1) == 1 ]]
```
to also keep it consistent with the rest of the shell
idioms used in this file, i.e. `$()`, `[[ ... ]]`, etc.

Reply:
Sorry, I was just trying things.

I will update it. Note that doing the `grep` matching `1/1` is not working because that matches `1/1       Terminating`.

I will try the other suggestion.

Reply:
> I will update it. Note that doing the grep matching 1/1 is not working because that matches 1/1 Terminating.

You could then try:
```
[[ $(kubectl get pod "${@}" | egrep -c "1/1.*Running") == 1 ]]
```